examples/burial-form/BurialInformation.jsx

```diff
@@ -52,7 +52,7 @@ export default function BurialInformation(props) {
           className="vads-u-border-color--primary-alt-light vads-u-border-left--4px vads-u-padding-left--2 vads-u-padding-y--0p5 vads-u-margin-left--neg2p5"
           name="locationOfDeath.other"
           label="Please specify where the Veteran's death occurred"
-          required={values.locationOfDeath.location === 'other'}
```

Comment:
this seems like it would like this would be important to the logic of the form. Maybe something like `values.locationOfDeath.location === 'other' && "A specific location of the Veteran's death is required"`?

Reply:
I haven't tested this so please correct me if I am wrong, but it seems like it would still throw an error even if `values.locationOfDeath.location === 'other'` isn't equal to true.

Reply:
@mlauren, nice catch! I've re-added these checks.